google-cloud-storage/src/main/java/com/google/cloud/storage/StorageException.java

```diff
@@ -66,7 +66,7 @@ public StorageException(GoogleJsonError error) {
    *
    * @throws StorageException when {@code ex} was caused by a {@code StorageException}
    */
-  static StorageException translateAndThrow(RetryHelperException ex) {
+  public static StorageException translateAndThrow(RetryHelperException ex) {
```

Comment:
I know why you need it, but it's a little strange to make this public. Maybe document that users normally wouldn't need to use this?

Reply:
`@InternalApi`'d